cluster/aws/templates/format-disks.sh

```diff
@@ -135,23 +135,9 @@ else
       move_kubelet="/mnt/ephemeral/kubernetes"
      else
       # aufs
-
-      # We used to split docker & kubernetes, but we no longer do that, because
-      # host volumes go into the kubernetes area, and it is otherwise very easy
-      # to fill up small volumes.
-
-      release=`lsb_release -c -s`
-      if [[ "${release}" != "wheezy" ]] ; then
-        lvcreate -l 100%FREE --thinpool pool-ephemeral vg-ephemeral
-
-        THINPOOL_SIZE=$(lvs vg-ephemeral/pool-ephemeral -o LV_SIZE --noheadings --units M --nosuffix)
-        lvcreate -V${THINPOOL_SIZE}M -T vg-ephemeral/pool-ephemeral -n ephemeral
-      else
-        # Thin provisioning not supported by Wheezy
-        echo "Detected wheezy; won't use LVM thin provisioning"
-        lvcreate -l 100%VG -n ephemeral vg-ephemeral
-      fi
-
+      # No need for thin pool since we are not over-provisioning or doing snapshots
```

Comment:
Maybe also write:

`We used to do thin provisioning, but this caused problems such as #16188.
No need for thin pool since we ....`